services/pull/merge.go

```diff
@@ -76,7 +82,8 @@ func Merge(pr *models.PullRequest, doer *models.User, baseGitRepo *git.Repositor
 	headRepoPath := pr.HeadRepo.RepoPath()
 
 	if err := git.InitRepository(tmpBasePath, false); err != nil {
-		return fmt.Errorf("git init: %v", err)
+		log.Error("git init tmpBasePath: %v", err)
+		return fmt.Errorf("git init tmpBasePath: %v", err)
```

Comment:
Other parts of this function returns just err
```suggestion
		return err
```